From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Franz-Josef Haider <franz.haider@jolla.com>
Date: Wed, 6 Jun 2018 09:28:27 +0200
Subject: [PATCH] (hybris) avoid attempting to mount partitions, mer/systemd
 handles this

Even partitions weren't mounted by this previously it still would produce an
error and further boot.

Change-Id: Icaea2cd8d145d8205f4a130119aedbd2720c868d
---
 init/builtins.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/init/builtins.cpp b/init/builtins.cpp
index a95a4a315b64afc1b87b40dca0ee20b8d732d744..c859cea33d4616934d82e4ca42af3102d61cb66f 100644
--- a/init/builtins.cpp
+++ b/init/builtins.cpp
@@ -633,6 +633,7 @@ static Result<void> do_mount_all(const BuiltinArguments& args) {
     std::string prop_name = "ro.boottime.init.mount_all."s + prop_post_fix;
     android::base::Timer t;
 
+#if 0
     Fstab fstab;
     if (mount_all->fstab_path.empty()) {
         if (!ReadDefaultFstab(&fstab)) {
@@ -645,6 +646,8 @@ static Result<void> do_mount_all(const BuiltinArguments& args) {
     }
 
     auto mount_fstab_result = fs_mgr_mount_all(&fstab, mount_all->mode);
+#endif
+    MountAllResult mount_fstab_result = {0,0};
     SetProperty(prop_name, std::to_string(t.duration().count()));
 
     if (mount_all->import_rc) {
@@ -660,6 +663,8 @@ static Result<void> do_mount_all(const BuiltinArguments& args) {
         initial_mount_fstab_return_code = mount_fstab_result.code;
     }
 
+    queue_event = false; // hybris does trigger_late_start.
+
     if (queue_event) {
         /* queue_fs_event will queue event based on mount_fstab return code
          * and return processed return code*/
